### ISSUE TITLE
[Bug -- Retail] Action[SetPoint] failed because[SetPoint would result in anchor family connection]: attempted from: ReputationDetailFrame:SetPoint.

### ISSUE THREAD

**Issue by JourneyOver** [siweia/NDui#268]
For the past day or two, I've been experiencing an issue where, upon logging into any character, the info bars in the top left and bottom right completely disappear, and the minimap reverts to the default Blizzard minimap. This problem persists until I do a `/reload`. Additionally, if I log out of the character and then back in, either to the same character or a different one, the UI is once again broken until I `/reload`.

Today, I finally had some time to sit down and test to see if it was an addon conflict or a bug with NDui. I ended up narrowing it down to the `Reskin Blizzard UI` option being enabled as what seems to be the cause of the issue.

Disabling the option resolves the issue, and everything works as expected upon logging in or switching characters. I have tried downgrading NDui to older versions to see if an unexpected addition might have caused the issue, but every version I've tried produces the same result and the same error message so I'm thinking that blizzard might of updated an api call or something as of very recent that could be causing this issue.

Full error message:
```
Action[SetPoint] failed because[SetPoint would result in anchor family connection]: attempted from: ReputationDetailFrame:SetPoint.
Count: 1

Call Stack:
[string "=[C]"]: ?
[string "=[C]"]: ?
[string "=[C]"]: in function `SetPoint'
[string "@Interface/AddOns/NDui/Modules/Skins/Blizzard/FrameXML/CharacterFrame.lua"]:259: in function `func'
[string "@Interface/AddOns/NDui/Modules/Skins/Skins.lua"]:33: in function `LoadAddOnSkins'
[string "@Interface/AddOns/NDui/Modules/Skins/Skins.lua"]:60: in function `OnLogin'
[string "@Interface/AddOns/NDui/Init.lua"]:136: in function `func'
[string "@Interface/AddOns/NDui/Init.lua"]:23: in function <Interface/AddOns/NDui/Init.lua:18>
```

**Comment by JourneyOver** [comment]
Looks like a temporary fix is just to go into `NDui/Modules/Skins/Blizzard/FrameXML/CharacterFrame.lua` and just comment out or remove line 259 (I can't tell if it breaks anything or not but things are looking fine for the most part). 

I did try the whole method of going and deleting the `layout-local.txt` for all my characters, and while it worked for the very first login of those characters after the deletion, if I logged out and back in again it would start throwing the issue again. So guess for now I'm just going to do my temp fix until this is resolved.

**Comment by siweia** [comment]
Does it occur with ndui enable only? The only posibility I can tell is, one of your addon modify the button anchors, which cause the conflicts.

**Comment by JourneyOver** [comment]
> Does it occur with ndui enable only? The only posibility I can tell is, one of your addon modify the button anchors, which cause the conflicts.

I can't remember if I tested with NDui only enabled or not, so I'll test again whenever the servers come back up.

**Comment by JourneyOver** [comment]
@siweia I'm not sure if it's just a coincidence or not, but it looks like this issue has been fixed in the 10.2.7 patch. At least, that's what I can tell, as it's not happening right now. I'm going to keep this open for now, though, and close it in a few days if this issue is truly resolved after a bunch of other addons are updated and such.

**Comment by JourneyOver** [comment]
Well so far this issue hasn't come up again the past couple days now, so I'm going to go ahead and close this. If it comes up again in the future I'll re-open this ticket.
